Picture.py

```diff
@@ -324,30 +402,29 @@ def crop(self, upperLeftX, upperLeftY, width, height):
         Picture
             the cropped copy of the original picture
         """
-        pic = self
-        pic.crop((upperLeftX, upperLeftY, upperLeftX+width, upperLeftY+height))
+        croppedImage = self.image.crop((upperLeftX, upperLeftY, upperLeftX+width, upperLeftY+height))
```

Comment:
Two things. First of all does this way of cropping actually change the pictures image. Secondly I was looking through the documentation for the original JES and it doesn't seem like crop is even supported.

Reply:
I think you're right, but the Picture class does have it so it does seem good to have.